.github/workflows/integration-tests-vm-type.yml

```diff
@@ -113,14 +119,24 @@ jobs:
           if [[ '${{ contains(github.event.pull_request.labels.*.name, 'integration-tests-trace-logging') }}' == 'true' ]]; then
             echo "COLLECTOR_LOG_LEVEL=trace" | tee -a "$GITHUB_ENV"
           fi
-
+          
           if [[ '${{ inputs.vm_type }}' =~ s390x ]]; then
             {
               echo "IC_API_KEY=${{ secrets.IBM_CLOUD_S390X_API_KEY }}"
               echo "IC_REGION=ca-tor"
             } >> "${GITHUB_ENV}"
           fi
 
+          if [[ '${{ inputs.vm_type }}' =~ ppc64le ]]; then
+            {
+              echo "IC_API_KEY=${{ secrets.IBM_CLOUD_POWER_API_KEY }}"
+              echo "IC_REGION=osa"
+              echo "IC_ZONE=osa21"
+            } >> "${GITHUB_ENV}"
+          
+            sudo ifconfig eth0 mtu 1000 up
```

Comment:
I need to have a think about how we can appropriately handle this. I think having it in its own step within the workflow is most appropriate, or we can move it into the ansible playbooks, but it should probably move from where it is, and requires some explanation for why it is necessary.

Reply:
I think this still stands - We should move this to its own step in the workflow, and only run that step when we're testing a power VM. We also need to make sure it is properly commented, ideally with links to the articles / bug tickets we've used to identify the issue

Reply:
I did not get actual place where needs to be add this line

Reply:
Essentially we need:

```
- name: Set MTU for Power VMs
  if: ${{ contains(inputs.vm_type, 'ppc64le') }}
  run: sudo ifconfig eth0 mtu 1000 up
```

Don't forget to include some comments about why it is necessary